betterdiscordctl

```diff
@@ -238,103 +188,65 @@ while :; do
   shift
 done
 
-mkdir -p "$data"
-[[ -f $data/bd_map ]] || touch "$data/bd_map"
-
 # Commands
 
 bdc_status() {
   index_mod=no
-  linked_dir=no
-  linked_repo=no
-  if [[ -d $core/injector ]]; then
-    if [[ -h $core/injector ]]; then
-      linked_dir=$(readlink "$core/injector")
-      if pushd "$core/injector" >/dev/null; then
-        linked_repo=$(git remote get-url origin 2>/dev/null || printf 'no\n')
-        popd >/dev/null
-      else
-        linked_dir="(broken link) $linked_dir"
-      fi
-    fi
-  fi
-  if [[ ! -f $core/index.js ]]; then
-    index_mod='(missing) no'
-  else
-    grep -q 'injector' "$core/index.js" && index_mod=yes
-  fi
+  grep -q betterdiscord.asar "$core/index.js" && index_mod=yes
```

Comment:
Quote static strings. `betterdiscord.asar` should never have a chance of being a glob here. Globbing is a different intent.